README.md

```diff
@@ -245,11 +245,17 @@
 ### Term representation
 
 * <a name="underscores-in-numerics"></a>
-  Add underscores to large numeric literals to improve their readability.
+  Add underscores to base-10 numeric literals that have six or more digits.
```

Comment:
"base-10 numeric literals" –> "decimal literals" (we use hex literals in other place)?